charts/hcloud-fip-controller/README.md

```diff
@@ -0,0 +1,118 @@
+hcloud-fip-controller
+-------
+
+hcloud-fip-controller is a small controller, to handle floating IP management in a [Kubernetes][k8s-home] cluster on hetzner cloud virtual machines.
+
+The running pods will check the hetzner cloud API every 30 seconds to check if the configured IP Addresses (or subnets in case of IPv6) are assigned to the server, the leader is scheduled to. If not it will update the assignments.
+You need to make sure, to have the IP Addresses configured on **every** node for this failover to correctly work, as the controller will not take care of the network configuration of the nodes.
+
+TL;DR;
+------
+
+```console
+$ helm repo add cbeneke https://cbeneke.github.com/helm-charts
+$ helm repo update
+$ helm install --name hcloud-fip-controller cbeneke/hcloud-fip-controller
+```
+
+Introduction
+------------
+
+This chart bootstraps a [hcloud-fip-controller][hcloud-fip-controller-home] installation on
+a [Kubernetes][k8s-home] cluster using the [Helm][helm-home] package manager.
+
+Prerequisites
+-------------
+
+-  Kubernetes 1.9+
+
+Installing the Chart
+--------------------
+
+The chart can be installed as follows:
+
+```console
+$ helm repo add cbeneke https://cbeneke.github.com/helm-charts
+$ helm repo update
+$ helm install --name hcloud-fip-controller cbeneke/hcloud-fip-controller
+```
+
+The command deploys hcloud-fip-controller on the Kubernetes cluster. This chart does
+not provide a default configuration; hcloud-fip-controller will not act on your
+Kubernetes Services until you provide
+one. The [configuration](#configuration) section lists various ways to
+provide this configuration.
+
+Uninstalling the Chart
+----------------------
+
+To uninstall/delete the `hcloud-fip-controller` deployment:
+
+```console
+$ helm delete hcloud-fip-controller
+```
+
+The command removes all the Kubernetes components associated with the
+chart, but will not remove the release metadata from `helm` — this will prevent
+you, for example, if you later try to create a release also named `hcloud-fip-controller`). To
+fully delete the release and release history, simply [include the `--purge`
+flag][helm-usage]:
+
+```console
+$ helm delete --purge hcloud-fip-controller
+```
+
+Configuration
+-------------
+
+See `values.yaml` for configuration notes. Specify each parameter
+using the `--set key=value[,key=value]` argument to `helm
+install`. For example,
+
+```console
+$ helm install --name hcloud-fip-controller \
+  --set rbac.create=false \
+    cbeneke/hcloud-fip-controller
+```
+
+The above command disables the use of RBAC rules.
+
+Alternatively, a YAML file that specifies the values for the above
+parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name hcloud-fip-controller -f values.yaml stable/hcloud-fip-controller
```

Comment:
this should be `cbeneke/hcloud-fip-controller` instead of `stable/hcloud-fip-controller`